lib/delta-types/src/Data/Delta.hs

```diff
@@ -5,32 +5,55 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 {- HLINT ignore "Use newtype instead of data" -}
-module Data.Delta (
-    -- * Synopsis
-    -- | Delta encodings.
-    --
-    -- The type constraint 'Delta'@ delta@ means that the type @delta@
-    -- is a delta encoding of the corresponding base type 'Base'@ delta@.
-    --
-    -- Delta encodings can be transformed into each other using an 'Embedding'.
 
-    -- * Delta encodings
+{-|
+Copyright: © 2021-2023 IOHK, 2024 Cardano Foundation
+License: Apache-2.0
+
+Delta types.
+
+The type constraint 'Delta'@ delta@ means that the type @delta@
+is a delta type of the corresponding base type 'Base'@ delta@.
+
+Delta types can be transformed into each other using an 'Embedding'.
+
+Examples:
+
+prop> apply (Replace 7) 3 = 7
```

Comment:
```suggestion
λ> apply (Replace 7) 3 = 7
```

Reply:
Actually, `prop>` is special markup for properties that Haddock supports. It will be rendered without the `prop>` prefix. 🤓 

https://haskell-haddock.readthedocs.io/en/latest/markup.html#properties

<img width="476" alt="haddock prop" src="https://github.com/cardano-foundation/cardano-wallet/assets/564592/8db5a301-2029-44f5-b7d6-452c83316409">